Lib/multiprocessing/managers.py

```diff
@@ -654,8 +687,8 @@ def register(cls, typeid, callable=None, proxytype=None, exposed=None,
 
         if method_to_typeid:
             for key, value in list(method_to_typeid.items()):
-                assert type(key) is str, '%r is not a string' % key
-                assert type(value) is str, '%r is not a string' % value
+                assert isinstance(key, str), '%r is not a string' % key
```

Comment:
Umm... OK. Someone was complaining that this wasn't done with the old 5001 patch...